### PR TITLE
W-14694395 - Split mule version to use monthly patches in the actual lib dependencies

### PR DESCRIPTION
**Pull request by elcoya**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -109,15 +109,16 @@
...
+        <mule.weave.version>2.1.2-20220125</mule.weave.version>
         <mule.sdk.version>1.1.5</mule.sdk.version>
         <mule.api.version>1.1.1</mule.api.version>
         <mule.metadata.version>1.1.1</mule.metadata.version>
         <mule.extensions.ast.loader.version>1.0.1</mule.extensions.ast.loader.version>
         <mule.extensions.maven.plugin.version>1.1.6</mule.extensions.maven.plugin.version>
         <mule.app.plugins.maven.plugin.version>1.1.1</mule.app.plugins.maven.plugin.version>
 
-        <guava.version>25.1-jre</guava.version>
+        <guava.version>32.0.0-android</guava.version>
```
Comment (nicomz) [inline pom.xml:121]: Why an Android Version?

**Review by nicomz — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (elcoya) [reply pom.xml]: My bad. It was the suggestion from dependabot, but there is a `jre` version available so changing it.

File: pom.xml [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
     <groupId>org.mule.extensions</groupId>
     <artifactId>mule-modules-parent</artifactId>
-    <version>1.1.11-SNAPSHOT</version>
+    <version>1.1.11</version>
```
Comment (elrodro83) [inline pom.xml:10]: do the proper release in its own separate pr

**Review by elrodro83 — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (elcoya) [reply pom.xml]: ok, done!

**Review by aiannucci — APPROVED** [review]

**Review by elrodro83 — APPROVED** [review]

**Pull request merged**